test/unit/math/prim/prob/neg_binomial_2_test.cpp

```diff
@@ -239,16 +239,64 @@ TEST(ProbDistributionsNegBinomial, chiSquareGoodnessFitTest4) {
 }
 
 TEST(ProbDistributionsNegBinomial, extreme_values) {
-  int N = 100;
-  double mu = 8;
-  double phi = 1e12;
-  for (int n = 0; n < 10; ++n) {
-    phi *= 10;
-    double logp = stan::math::neg_binomial_2_log<false>(N, mu, phi);
-    EXPECT_LT(logp, 0);
+  std::array<unsigned int, 5> n_to_test = {1, 5, 100, 12985, 1968422};
```

Comment:
[optional]

```
std::vector<unsigned int> n_to_test = ...
```
is shorter and easier to modify.  And speed won't matter here in the tests.

Reply:
Sure, I kind of missed you can do this with `std::vector`. Modified everywhere.